### DISCUSSION TITLE
Under very specific circumstances, the first column in a document is not printed.

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#10] [1.2 Bugs and Issues]
Dear Alex, dear community,

I think that the following is a very curious bug concerning printing, namely that the first column in a document is not printed.
I do not know exactly when it occurs, but I could find one setting that causes it.

Choosing ASCII:INI as filter:
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/c254baa4-b47b-4b71-96b3-ed030a501dc6)
The options Wrap Long Lines and Wrap Environments are chosen:
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/e71d0798-e035-47bd-8268-be2ed5258a9f)
Printing to PDF (actually, it also happened with a real printer)
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/199da2c2-08a9-46f2-88fd-c687077ed91d)
The first column is missing:
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/0393e437-4158-4206-8881-c795d9965c9e)
It does not seem to happen if the mode is TeX.
Also if Wrap Environment is chosen without Wrap Long Lines or if Wrap Long Lines without Wrap Environment is chosen alone, it does not seem to happen.

WinEdt Build: 20230519 (v. 11.1) - 64-bit  
Operating system: Microsoft Windows 11 Home (10.0.22631 Build 22631)

Best,
Thomas

**Comment by Alex-Simonic** [comment]
This "bug" has been there for a long time. It does not affect softly wrapped documents. Perhaps nobody else attempted to print a "raw data" document with wrapping.

Anyway, it has now been fixed.
